src/Chapter2.hs

```diff
@@ -605,7 +624,8 @@ Implement a function that duplicates each element of the list
 
 -}
 duplicate :: [a] -> [a]
-duplicate = error "duplicate: Not implemented!"
+duplicate [] = []
+duplicate (x:xs) = x : x : duplicate(xs)
```

Comment:
No need to take argument in the parentheses 🙂 
Also, in Haskell, functions are always separated from arguments by spaces 👾 

```suggestion
duplicate (x:xs) = x : x : duplicate xs
```